packages/core/src/ctype/CType.spec.ts

```diff
@@ -198,7 +132,7 @@ describe('blank ctypes', () => {
 
 describe('CType verification', () => {
   const ctypeInput = {
-    $id: 'kilt:ctype:0x1',
+    $id: 'kilt:ctype:0xdeadbeef',
```

Comment:
As a side note, in my code I try to avoid words with dark meaning :) I was shocked seeing someone putting `0xdeaddad1` somewhere.

Reply:
there is not really a reason to change it from what it was, I think that's just because I had removed that id altogether and then added one again